config.go

```diff
@@ -443,8 +469,10 @@ type Config struct {
 	//
 	// Optional: defaults to 200
 	DataIterationBatchSize uint64
-	// Optional: Map of schemaName(source schema) => tableName => uint64 to override default values for certain tables
-	DataIterationBatchSizePerTableOverride map[string]map[string]uint64
```

Comment:
Note: we will need to update the config to use the `DataIterationBatchSizePerTableOverride` struct when we bump the ghostferry version